### ISSUE TITLE
Triple-quoted strings are incorrectly indented in _ren.py files

### ISSUE THREAD

**Issue by eevee** [renpy/renpy#5525]
This:

```
"""renpy
init python:
"""

a = """foo
bar"""
```

is transformed into this:

```
init python:
    a = """foo
    bar"""
```

which changes the contents of `a`.

Took me a while to realize where the extra space was coming from.  That said, this doesn't look pleasant to fix, either.

**Comment by Gouvernathor** [comment]
That's how it is intended and [documented](https://www.renpy.org/doc/html/ren_py.html) to behave.

**Comment by renpytom** [comment]
Sure, but it's also kinda bad - and wasn't intended. I need to think about if a fix is possible.

**Comment by Gouvernathor** [comment]
That wouldn't be easy.
You need to flag the lines that are between opening triple quotes (exclusive) and closing triple quotes (inclusive) ; where no quote is protected by backslash ; not included in a string of the other type of quote (double vs single) ; and that aren't in a comment.

**Comment by renpytom** [comment]
Yes, but we do this in other parts of Ren'Py's parser, so we could do it here.

**Comment by eevee** [comment]
It's actually even more not-easy — you can escape newlines in single-quoted strings, so the line after that shouldn't be indented either.  And f-strings can contain arbitrary Python expressions, including the same quotes again, as of 3.12.

I suspect it could still be done without the full Python parser, with a Sufficiently Clever™ regex, but it may not be worth it.

**Comment by Andykl** [comment]
Do we really need to keep current behavior of ren.py files? 
I propose we do
```
Ignore section is still ignored

"""renpy

Here code is the same as rpy

[init prio] python [early] [in namespace]:
"""
Code here is parsed as ast.Python but not indented

"""renpy

any other statement:
"""
Error: Can't switch to Python using "any other statement"
```

That way this issue is solved, and we can drop current unindenting of traceback lines. 

I don't think anyone used non init python or python early as block marker, because it's either invalid renpy or invalid python code.
